container/build.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+import argparse
+import os
+import subprocess
+
+# Map to store all supported container type and the package of target to build it.
+TYPE_PACKAGE_MAP = {
+    "rbe-debian8": "container/debian8/builders/rbe-debian8",
```

Comment:
lets create a list or something to store the names of all the containers used in all these maps

Reply:
I added a list of the supported types, and a check to make sure the dictionaries's keys match that list. Let me know if you had something else in mind.